### ISSUE TITLE
iOS UI customization in monorepo project structure

### ISSUE THREAD

**Issue by Alieltahan** [Adyen/adyen-react-native#292]
**Describe the bug**
After following the docs, I applied the UI customization for Android ( and it worked as expected ) for iOS not styles not reflected.

**To Reproduce**
App Structure: ("apps/{APPNAME}/ios/{TENANT}/AdyenAppearance.swift")

**Expected behavior**
iOS styling to be updated.

**Additional context**
I applied the same UI styling in the AdyenExample repo and worked fine.
I have monorepo project. with multi tenant.
The DropIn component runs fine.

**Comment by descorp** [comment]
Hey @Alieltahan 

`AdyenAppearance.swift` need to be added in Xcode. Also make sure to set correct "Target Membership" to it.

---

To debug this make a breakpoint in [AdyenApperance](https://github.com/Adyen/adyen-react-native/blob/develop/ios/Components/AdyenApperance.swift#L19C16-L19C37) and in Xcode console log and print out 
```
po Bundle.allBundles.compactMap { $0.infoDictionary?[bundleExecutableKey] as? String }
```

**Comment by Alieltahan** [comment]
Thank you @descorp  for your support, Kindly note:
- I had added the file in Xcode, and selected a Target Membership.
-  I'm facing the following error while trying to print:

<img width="848" alt="image" src="https://github.com/Adyen/adyen-react-native/assets/46691446/ce3f1567-bad4-4bd0-bc03-92b1d53dd289">

Appreciate your feedback.

**Comment by descorp** [comment]
@Alieltahan 

try 
```swift
Bundle.allBundles.compactMap { $0.infoDictionary?["CFBundleExecutable"] as? String }
```

---

However, since no style is applied, I'm guessing this code won't be reached either.

What you need to do is to find file `AdyenApperance.swift` in Xcode and make a breakpoint. 
Should be somewhere in `Pods/adyen-react-native`

You should be able to do so by typing the name in Xcode file filter (see example below):
![Screenshot 2023-10-18 at 16 57 52](https://github.com/Adyen/adyen-react-native/assets/2648655/69c25684-03b2-44c0-b851-cd0564f66228)

**Comment by descorp** [comment]
Hey @Alieltahan 

Were you able to resolve the issue?

**Comment by Alieltahan** [comment]
@descorp Thank you for your follow up.
I'm still unable to resolve it at the moment.

Please find the output:
`["App-TE", "ContactViewViewService", "BoundingPathData"]`

Appreciate your support.

**Comment by descorp** [comment]
Hey @Alieltahan 

I think "-" in your target name might be a cause of the problem.

Here is PR to fix this #300
Could you try this fix locally by updating "AdyenAppearance.swift" in **Pods**  ?

**Comment by Alieltahan** [comment]
Hello @descorp 
This is worked perfect!

Thank you so much for your extended support.

Have a great day ahead.

**Comment by descorp** [comment]
Thank you for confirming 💚
I'll keep this ticket open till we fix the bug in 1.2.0 release

**Comment by descorp** [comment]
Release on [1.2.0](https://github.com/Adyen/adyen-react-native/releases/tag/1.2.0)
